docs/sources/upgrading/_index.md

```diff
@@ -33,6 +33,31 @@ The output is incredibly verbose as it shows the entire internal config struct u
 
 ## Main / Unreleased
 
+### Loki
+
+#### Default retention_period has changed
+
+This change will affect you if you have:
+```yaml
+compactor:
+  retention_enabled: true
+```
+
+And did *not* define a `retention_period` in `limits_config`, thus relying on the previous default of `744h`
+
+In this release the default has been changed to `0s`.
+
+A value of `0s` is the same as "retain forever" or "disable retention".
```

Comment:
> A value of `0s` is the same as "retain forever" or "disable retention".

I think it's worth adding this into our config doc as well.

Reply:
we do have ` As of version 2.8.0, a zero value of 0 or 0s disables retention.` in the config section already, seems like this should be good enough?